xbmc/interfaces/json-rpc/DatabaseOperations.h

```diff
@@ -0,0 +1,26 @@
+/*
+ *  Copyright (C) 2025 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#pragma once
+
+#include "JSONRPC.h"
```

Comment:
Please also add 
```c++
#include <string>
```
after this include (order is important, otherwise clang-format will moan).